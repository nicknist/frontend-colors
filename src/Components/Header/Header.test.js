import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  })
})
