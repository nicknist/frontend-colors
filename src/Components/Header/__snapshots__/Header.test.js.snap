// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <p>
    Colin and Nick's Color Party
  </p>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
</header>
`;
